### PR TITLE
refactor!: `Chip` move to kit

### PR DESCRIPTION
**Pull request by vladimirpotekhin**
Closes #

### CODE REVIEW

**Review by splincode — APPROVED** [review]

File: projects/demo/src/modules/components/tag/tag.template.html [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <tui-doc-page
+    deprecated
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/tag/tag.template.html:2]: Will it be deleted in separate PR ?

**Review by nsbarsukov — APPROVED** [review]

File: projects/demo/src/modules/components/tag/tag.template.html (hunk 1 above)
Comment (vladimirpotekhin) [reply projects/demo/src/modules/components/tag/tag.template.html:2]: yes, after replacing tags to chips in `multiselect` / `input-tag`. After new textfields, probably
